### ISSUE TITLE
Adding an arguement to mxnet.symbol.softmax()

### ISSUE THREAD

**Issue by opringle** [apache/incubator-mxnet#9473]
- I have an mxnet.module with a custom loss function
- As a result I need to create a separate module from an internal symbol to get the model predictions
- This symbol is an mxnet.symbol.softmax() layer
- Because the symbol does not have label as an input, I cannot use mxnet.metric methods with it

Can anyone advise me as to how to get mxnet.metric api working with internal model symbols? Do I just need to add label as an arguement to the symbol?

**Comment by opringle** [comment]
Solution is here: https://github.com/apache/incubator-mxnet/issues/5229
